src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -123,6 +125,28 @@ public void resetOdometry(Pose2d pose) {
         pose);
   }
 
+  /**
+   * Correction for swerve second order dynamics issue. Borrowed from 254:
+   * https://github.com/Team254/FRC-2022-Public/blob/main/src/main/java/com/team254/frc2022/subsystems/Drive.java#L325
+   * Discussion:
+   * https://www.chiefdelphi.com/t/whitepaper-swerve-drive-skew-and-second-order-kinematics/416964
+   */
+  private static ChassisSpeeds correctForDynamics(ChassisSpeeds originalSpeeds) {
+    final double LOOP_TIME_S = 0.02;
```

Comment:
Where does this number come from?

Reply:
And should it be in `Cal.java`, or stay here?

Reply:
This is the basic rate that all the periodics/everything execute at, we can pretty much guarantee it'll never change